standard/classes.md

```diff
@@ -3050,8 +3050,8 @@ class Customer
 
 When the first parameter of a method includes the `this` modifier, that method is said to be an ***extension method***. Extension methods shall only be declared in non-generic, non-nested static classes. The first parameter of an extension method is restricted, as follows:
 
-- It may have the parameter modifier `in` only if the parameter has a value type
-- It may have the parameter modifier `ref` only if the parameter has a value type or is a generic type constrained to struct
+- It may be an input parameter if the parameter has a value type
+- It may be be a reference parameter if the parameter has a value type or is a generic type constrained to struct
```

Comment:
As previous but also remove the “be be” typo:

```suggestion
- It may only be a reference parameter if it has a value type or has a generic type constrained to struct
```